src/test/scala/cognite/spark/v1/RawTableRelationTest.scala

```diff
@@ -755,4 +755,39 @@ class RawTableRelationTest
     err.getMessage shouldBe "Error while loading RAW row [key='k'] in column 'value': java.lang.NumberFormatException: For input string: \"test\""
 
   }
+
+  private def inferSchema(json: String): DataType = {
+    val circeJson = io.circe.parser.parse(json).right.get
+    RawSchemaInferrer.toSparkSchema(RawSchemaInferrer.infer(circeJson))
+  }
+
+  "RawSchemaInferrer" should "handle integers" in {
```

Comment:
Maybe we can get some additional good test cases from Spark's `JsonInferSchema`?